public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -15,24 +15,36 @@ const options: monacoTypes.editor.IStandaloneEditorConstructionOptions = {
   fixedOverflowWidgets: true,
   folding: false,
   fontSize: 14,
-  lineDecorationsWidth: 8,
+  lineDecorationsWidth: 8, // used as "padding-left"
   lineNumbers: 'off',
   minimap: { enabled: false },
   overviewRulerBorder: false,
   overviewRulerLanes: 0,
   padding: {
+    // these numbers were picked so that visually this matches the previous version
+    // of the query-editor the best
     top: 4,
-    bottom: 4,
+    bottom: 5,
   },
   renderLineHighlight: 'none',
   scrollbar: {
     vertical: 'hidden',
+    verticalScrollbarSize: 8, // used as "padding-right"
+    horizontal: 'hidden',
+    horizontalScrollbarSize: 0,
   },
   scrollBeyondLastLine: false,
   suggestFontSize: 12,
-  wordWrap: 'off',
+  wordWrap: 'on',
 };
 
+// this number was chosen by testing various values. it needs to do 2 things:
```

Comment:
Is it because `editor.getContentHeight();` doesn't include borders?

Reply:
could be... i added it to the comments.